test/testfiles/test_cube_unstructured_2.inp

```diff
@@ -0,0 +1,244 @@
+*Heading
+ C:\Users\Erik\Downloads\gmsh-4.8.4-Windows64\cube_unstructured_2.inp
```

Comment:
Is it necessary, or can the filepath be removed?

Reply:
Removed. Done.